RELEASE.md

```diff
@@ -1,3 +1,40 @@
+# Release 2.2.0
+
+The 2.2 minor series tracks TensorFlow 2.2.
+
+## Features
+
+- Profile plugin now should be pip installed.
+- Debugger:
+  - Added support for alerts (#3285)
+  - Support FULL_HEALTH debug mode (#3343)
+  - Integration with ngrx storage (#3359)
+- Add S3_ENDPOINT variable (#3347)
+- Confirm that the connection to tensorboard works or change to localhost (#2371)
+- Update --reload_multifile_inactive_secs default to 24 hours (#3243)
+
+## TensorBoard.dev updates
+
+- Improvements to [TensorBoard.dev] support:
+  - New `tensorboard dev update-metadata` command allows for updating the name and
+    description of experiments (#3277)
```

Comment:
We’ve henceforth used the “TensorBoard.dev updates” section to describe
changes that occur at the same time as the release but are actually only
server-side changes and thus won’t appear in the Pip package. The new
`update-metadata` subcommand is actually in this release, so can go in
the features section above.

Likewise for the #3307 entry below.

Reply:
Done.